pages/order-list/index.js

```diff
@@ -1 +1,20 @@
-export default function OrderList() {}
+import Drinks from "/components/order-menu/drink-menu";
+import Maindishes from "/components/order-menu/main-menu";
+
+export default function OrderList() {
+  return (
+    <>
+      <h1>Order Menu</h1>
+      <form>
+        <ul>
+          <h2>Hauptspeisen</h2>
+          <Maindishes />
+        </ul>
+        <ul>
+          <h2>Getränke</h2>
```

Comment:
h1, h2 have no business in a ul as well

Reply:
> h1, h2 have no business in a ul as well

what should I use instead? leave it outside of the ul?